.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+language: java
+services:
+  - docker
+jdk:
+  - oraclejdk8
+os:
+  - linux
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+before_install:
+  - sudo apt-get install jq
+  - wget -O ~/codacy-coverage-reporter-assembly-latest.jar $(curl https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r .assets[0].browser_download_url)
+  - docker build -t "draft-store:pgsql" ./docker/database/
```

Comment:
is docker-compose not available on travis?

Reply:
it is but I think there are challenges then with pulling out artefacts like coverage reports. This way it's just the db inside a container and it's super simples to grab the bits I need to ship to codacy